README.md

```diff
@@ -18,4 +18,68 @@ Install the packages
 
 ### Testing the code
 
-`node src/index.js`
\ No newline at end of file
+`node src/index.js`
+
+### Allowed inputs
+
+* Select
+* Submit
+* Label
+* Reset
+* Email
+* Text
+* Password
+
+### Input Examples
+
+#### Select Input
+```
+{type: 'select', label: 'Gender', name: 'gender', options: [
+        {value: 'male', label: 'Male'},
+        {value: 'female', label: 'Female'}
+ ]}
+
+```
+### Output
+```
+<select name="gender">
+<option value="male">Male</option>
+<option value="female">Female</option>
+</select>
+```
+### Label & Text Input
+```
+{type: 'text', label: 'First Name', name: 'firstName', value: 'Enter your first name' }
+```
+### Output
+```
+<label>Last Name <input type="text" name="lastName" value="Enter your last name"/></label>
+```
+### Submit Input
+```
+{type: 'submit', value: 'Submit'}
+```
+### Output
+```
+<input type="submit" value="Submit">
+```
+### Reset Input
+```
+{type: 'reset', value: 'Reset'}
+```
+### Output
+```
+<input type="reset" value="Reset">
+```
+###Output
+```
+<input type="submit" value="Submit">
+```
+### Email Input
+```
+{type: 'text', label: 'Last Name', name: 'lastName', value: 'Enter your last name'}
+```
+### Output
+```
+<label>Email <input type="email" name="email" value="Enter your email"/></label>
+```
```

Comment:
Adding some additional documentation to give someone a better idea on how they should go about creating the inputs.

Reply:
Thank you for this it will make things look better